homeassistant/components/dremel_3d_printer/sensor.py

```diff
@@ -56,11 +56,11 @@ class Dremel3DPrinterSensorEntityDescription(
     ),
     Dremel3DPrinterSensorEntityDescription(
         key="remaining_time",
```

Comment:
my only thought is should the key change too?

Reply:
No only because the supporting package supports that key for this piece of data. It is how it is presented in the API.